src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBody.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.legaladvisor;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDHearingDurationType;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDOrderDirectionType;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDDirectionPartyType;
+import uk.gov.hmcts.cmc.claimstore.idam.models.UserDetails;
+import uk.gov.hmcts.reform.docassembly.domain.FormPayload;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Builder
+@EqualsAndHashCode
+public class DocAssemblyTemplateBody implements FormPayload {
+
+    private static final String HMCTS_URL = "https://www.gov.uk/make-money-claim";
+    private static final String COURT_SEAL_IMG = "[userImage:courtseal.PNG]";
+
+    private final int rowNum = 0;
+
+    @JsonProperty("courtseal")
+    private final String courtSeal = COURT_SEAL_IMG;
+
+    private final boolean displayComments = false;
+
+    private final String hmctsURL = HMCTS_URL;
+
+    @JsonProperty("displaycode")
+    private final int displayCode = 0;
+
+    private String referenceNumber;
+
+    private Party claimant;
+
+    private Party defendant;
+
+    private Judicial judicial;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
+    private LocalDate currentDate;
+
+    @JsonProperty("orderDirections1isIncluded")
+    private boolean hasFirstOrderDirections;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
+    private LocalDate docUploadDeadline;
+
+    @JsonProperty("orderDirections2isIncluded")
+    private boolean hasSecondOrderDirections;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
+    private LocalDate eyewitnessUploadDeadline;
+
+    @JsonProperty("hearingisRequired")
+    private boolean hearingIsRequired;
+
+    private CCDDirectionPartyType docUploadForParty;
+
+    private CCDDirectionPartyType eyewitnessUploadForParty;
+
+    private String preferredCourtName;
+
+    private String preferredCourtAddress;
+
+    private CCDHearingDurationType estimatedHearingDuration;
+
+    private String hearingStatement;
+
+    private List<OrderDirection> otherDirectionList;
+
+    @Builder
+    @EqualsAndHashCode
```

Comment:
can we externalize these Party to separate class files?

Reply:
sure! :)